src/EA.Weee.DataAccess/DataAccess/EvidenceDataAccess.cs

```diff
@@ -322,5 +322,18 @@ public Task<bool> HasApprovedWasteHouseHoldEvidence(Guid recipientId, int compli
                                n.Status.Value == NoteStatus.Approved.Value &&
                                n.WasteType.Value == WasteType.HouseHold);
         }
+
+        public Note DeleteZeroTonnageFromSubmittedTransferNote(Note note, NoteStatus status, NoteType type)
+        {
+            if (status.Equals(NoteStatus.Submitted) && type.Equals(NoteType.TransferNote))
+            {
+                var itemsToRemove =
+                    note.NoteTransferTonnage.Where(ntt => (ntt.Received == 0.00m && ntt.Reused == 0.00m) || (ntt.Received == null && ntt.Reused == null));
```

Comment:
@NateHogan  Wouldnt bother checking the reused values. If the Recieved is null or zero just remove as reused can't have value if received doesnt

Reply:
Updated